lms/djangoapps/certificates/api.py

```diff
@@ -91,6 +91,42 @@ def get_certificates_for_user(username):
     ]
 
 
+def get_certificate_for_user(username, course_key):
```

Comment:
The other function has a bug (feature) that I removed and I do not want to touch it. Plus I do not want to write tests for the other function (there are no tests at the moment).